### PR TITLE
spectrum_cass.xml: Software list additions

### PR DESCRIPTION
**Pull request by ArcadeShadow**
New working software list additions
--------------------------------------------
Bingo Parlante! [Luís Rato, Planeta Sinclair]
Casanova (Pizza e Pasta) [Planeta Sinclair]
Chess - The Turk (AABC Soft) [Spectrum Computing]
Em Busca do Mortadela (Portuguese) [Planeta Sinclair]
En Busca de Mortadelo (Spanish) [Planeta Sinclair]
Firebirds (Tronk) [Steven Brown, Planeta Sinclair]
Freez' Bees (Aackosoft) [Spectrum Computing]
Fun Golf [Luís Rato, Planeta Sinclair]
Gremlins 2: A Nova Geração [Planeta Sinclair]
Journey to the Centre of the Earth - Extended Version (10 Anniversary, English) [Topo Siglo XXI]
Lobo Solitário [João Cruz, Planeta Sinclair]
Lorna (Portuguese) [Planeta Sinclair]
Los Tesoros Perdidos de Tulum (v1.0, Spanish) [Spectrum Computing]
Los Tesoros Perdidos de Tulum (v1.1, Spanish) [RetroWorks]
Los Amores de Brunilda (v1.4, English) [RetroWorks]
Los Amores de Brunilda (v1.4, Spanish) [RetroWorks]
Los Amores de Brunilda (v1.4, Galician) [RetroWorks]
Los Amores de Brunilda (v1.4, Catalan) [RetroWorks]
Los Amores de Brunilda (v1.4, Catalan, alt) [RetroWorks]
Los Amores de Brunilda (v1.4, Basque) [RetroWorks]
Los Amores de Brunilda (v1.4, Italian) [RetroWorks]
Los Amores de Brunilda (v1.4, Russian) [RetroWorks]
Mad Mix 2: No Castelo dos Fantasmas (Portuguese) [Planeta Sinclair]
Masterfile v08 (Logiciels) [Spectrum Computing]
Masterfile (Plansoft) [Spectrum Computing]
R.A.M. (Portuguese) [Planeta Sinclair]
Schach [Spectrum Computing]
Searching for Mort (English) [Planeta Sinclair]
Shadow Warriors (Musical) [Spectrum Computing]
Slippery Sid (pirate) [Steven Brown, Planeta Sinclair]
Software Magazine №9A - Gold Mine [Spectrum Computing]
Software Magazine №11B - Patin-Man [Spectrum Computing]
Software Magazine №12B - Pincel-3D [Spectrum Computing]
Software Magazine №13A - The Guardian [Spectrum Computing]
Software Magazine №13B - Ghost Hunt (alt) [Spectrum Computing]
Super Scacchi (pirate) [Steven Brown, Planeta Sinclair]
The Lost Treasures of Tulum (v1.0, English) [Spectrum Computing]
The Sword of Ianna [RetroWorks]
Time-Gate (pirate) [Steven Brown, Planeta Sinclair]
Topo Mix Game (Portuguese) [Planeta Sinclair]
Toad Runner (Dro Soft) [Steven Brown, Planeta Sinclair]
Uccidi Gli Spiriti (pirate) [Steven Brown, Planeta Sinclair]
Viagem ao Centro da Terra - Versão Estendida (10 Aniversário, Portuguese) [Topo Siglo XXI]
Wonderful Dizzy (v1.5) [Team Yolkfolk]
Wonderful Dizzy (v1.0) [Team Yolkfolk]
Yatzy [Steven Brown, Planeta Sinclair]
Zip-Zap (pirate) [Steven Brown, Planeta Sinclair]
Zip-Zap (PRISM) [Spectrum Computing]
Zona 0 (Portuguese) [Planeta Sinclair]
Zynaps (IBSA) [Spectrum Computing]

Fixed Parent/CloneOf relationships, Publishers, Years and added Notes (information provided by spectrumcomputing.co.uk)

### CODE REVIEW

File: hash/spectrum_cass.xml [hunk 1]
```diff
@@ -6119,16 +6114,16 @@ no loading screen, no custom loader
...
-		<publisher>Magic Bytes / Dro Soft</publisher>
+		<publisher>Dro Soft</publisher>
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="50304">
 				<rom name="mortadelo y filemon - small case (dro soft).tzx" size="50304" crc="3079e2e2" sha1="4820dd28964c6fca6998822b8dcb47d14b9679c5"/>
 			</dataarea>
 		</part>
 	</software>
-
+
```
Comment (cuavas) [inline hash/spectrum_cass.xml:6126]: Stray whitespace here.  Please run srcclean on this file to reduce churn.

Comment (ArcadeShadow) [reply hash/spectrum_cass.xml]: Done it.

**Comment by cuavas** [comment]
Was promoting “Booty - The Remake (v1.2, TZX tape image)” to working intentional?  It isn’t noted.

**Comment by ArcadeShadow** [comment]
> Was promoting “Booty - The Remake (v1.2, TZX tape image)” to working intentional? It isn’t noted.

Yes it was intentional.
Sorry for the late reply.

**Pull request merged**